app/layout.tsx

```diff
@@ -6,8 +6,12 @@ import './globals.css';
 import '@swapr/ui/colors.css';
 
 import { Providers } from '@/providers';
-import { Navbar, Footer } from '@/app/components/ui';
+
+// exporting from components directory will create an error
```

Comment:
why?

and why do we need this comment?

Reply:
Fixed it, missing "use client" in some components led to errors